pkg/drivers/kic/oci/network_create.go

```diff
@@ -48,9 +49,15 @@ func CreateNetwork(ociBin string, name string) (net.IP, error) {
 	return createDockerNetwork(name)
 }
 
-func createDockerNetwork(clusterName string) (net.IP, error) {
+func createDockerNetwork(networkName string) (net.IP, error) {
+	// check if the network is docker native, if this is the case return
+	if networkName == "bridge" {
```

Comment:
use the contantant that we have for this
```
const dockerDefaultBridge = "bridge"
```

Reply:
thanks